operators/kas-strimzi-bundle-template.yaml

```diff
@@ -30,6 +32,14 @@ objects:
     name: kas-strimzi-bundle
     source: kas-strimzi-catalog
     sourceNamespace: ${NAMESPACE}
+    config:
+      resources:
+        requests:
+          memory: "1Gi"
+          cpu: "500m"
+        limits:
+          memory: "1Gi"
+          cpu: "500m"
```

Comment:
Once we solve https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/161/files/089afee335a31701ecd840b9ec94f689695f3af3#r859221284, this maybe solved as well